datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/validate_snmp_profiles.py

```diff
@@ -1,21 +1,79 @@
+import glob
```

Comment:
Do we support `match` keyword as described here ? https://datadoghq.dev/integrations-core/tutorials/snmp/profile-format/#metric_tags

We don't use it in our profiles, but some customers do.